play-json/jvm/src/main/scala/play/api/libs/json/EnvReads.scala

```diff
@@ -17,287 +17,41 @@ import java.time.{
 import java.time.format.{ DateTimeFormatter, DateTimeParseException }
 import java.time.temporal.UnsupportedTemporalTypeException
```

Comment:
Rename and keep in the new `EnvReads` only the JVM specifics instances for `java.time` and Jackson types